test/test_distributions.py

```diff
@@ -391,6 +399,24 @@ def get_sp_dist(jax_dist):
     T(dist.LogUniform, 1.0, 2.0),
     T(dist.LogUniform, 1.0, np.array([2.0, 3.0])),
     T(dist.LogUniform, np.array([1.0, 2.0]), np.array([[3.0], [4.0]])),
+    T(
+        dist.MatrixNormal,
+        1.0 * np.arange(6).reshape(3, 2),
+        np.array([[1.0, 0], [0.4, 1]]),
+        np.array([[1.0, 0, 0], [0.3, 0.36, 0], [0.4, 0.49, 4]]),
+    ),
+    T(
+        dist.MatrixNormal,
+        1.0 * np.arange(12).reshape((2, 3, 2)),
```

Comment:
Maybe we use `np.arange(6.).reshape((2, 3))` to check if broadcasting logic for `loc` is working?

Reply:
This is on me. I changed the order of `scale_tril_row` and `scale_tril_column` in `__init__` to match Wikipedia but forgot to changes this in `test_distribution.py`.